backend/src/main/java/ca/bc/gov/backendstartapi/entity/FavoriteActivityEntity.java

```diff
@@ -36,7 +39,8 @@ public class FavoriteActivityEntity {
   private UserEntity user;
```

Comment:
This property is now mapped to the wrong entity, as the table `spar.favorite_activity` doesn't have a foreign key to `spar.user` anymore.